uefi/flashregionsection.go

```diff
@@ -10,29 +10,44 @@ import (
 // FlashRegionSectionSize is the size of the Region descriptor. It is made up by 18 fields, each 16-bits large.
 const FlashRegionSectionSize = 36
 
+// Region contains a the start and end of a region in flash. This can be a BIOS, ME, PDR or GBE region.
+type Region struct {
+	Base  uint16
+	Limit uint16
+}
+
+// Available checks to see if a region is valid (has nonzero limit)
```

Comment:
I would take out the (has nonzero limit) bit because this function could end up doing more later.

Reply:
ok